hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -171,6 +115,8 @@ checkAbort context@ScriptContext{scriptContextTxInfo = txInfo} headContext parti
 --
 --   * The transaction is performed (i.e. signed) by one of the head participants
 --
+--   * ST token is present in the output
```

Comment:
ST already means token. I updated the description of all PT / ST mentions in the code in  82c0e75a1bc11d20178d639c32b95d42288796ed